spec/abstract/RISCV64/Init_A.thy

```diff
@@ -54,18 +62,25 @@ definition init_arch_state :: arch_state
      riscv_kernel_vspace = init_vspace_uses
    \<rparr>"
 
+definition toplevel_bits :: nat
+  where
+  "toplevel_bits = pt_bits_left max_pt_level"
+
+definition elf_index :: pt_index
+  where
+  "elf_index = ucast (pt_index max_pt_level kernel_elf_base)"
 
-(* {pptr_base ..< pptr_base + (1 << 30)} is pt index 0x100 at max_pt_level,
-   {kernel_elf_base ..< kernel_elf_base + (1 << 20)} comes out to pt index 0x1FE.
+(* {pptr_base ..< pptr_base + (1 << 20)} is pt index 0x100 at max_pt_level,
```

Comment:
However, it looks like I missed at least one instance. I guess this comment should be changed to use `kernel_window_bits` as well.